packages/frontend/src/contexts/game/actions/index.test.js

```diff
@@ -13,11 +14,11 @@ describe('GameContext actions', () => {
   });
 
   it(`returns an action object with type ${types.FLIP_TILE} and a payload containing a tile object`, () => {
-    const tile = mockPhotos[0];
+    const tile = createTilesFromPhotos(mockPhotos, {shuffle: false})[0]
 
     expect(flipTile(tile)).toStrictEqual({
       type: types.FLIP_TILE,
-      payload: { tile },
+      payload: {tile},
```

Comment:
linting rules should require spaces before and after curly brackets and also semi-colons at the ends of lines.